package.json

```diff
@@ -1,24 +1,27 @@
 {
   "name": "rebilly-js-sdk",
-  "version": "39.0.0",
+  "version": "40.0.0",
   "description": "Official Rebilly API JS library for the browser and Node",
   "browser": "./dist/rebilly-js-sdk.js",
   "main": "./dist/rebilly-js-sdk.node.js",
-  "types": "./dist/types/rebilly-js-sdk/index.d.ts",
+  "types": "./dist/rebilly-js-sdk.d.ts",
   "scripts": {
     "clean": "rimraf dist && rimraf coverage",
-    "build:prod": "npm run clean && webpack --env=prod",
-    "build:dev": "npm run clean && webpack --env=dev",
-    "prepublish": "npm run build:prod",
+    "build:prod": "npm run clean && webpack --env=prod && yarn ts:bundle-types-from-redocly",
+    "build:dev": "npm run clean && webpack --env=dev && yarn ts:bundle-types-from-redocly",
+    "prepublishOnly": "npm run build:prod",
     "test:unit": "jest",
     "test:unit:coverage": "jest --collect-coverage --no-cache",
     "test:unit:debug": "node --inspect-brk jest --no-cache --runInBand",
     "test:unit:watch": "jest --watchAll",
     "test": "npm run test:unit",
-    "ts:generate-sdk-types": "node ./scripts/type-generation/generate-sdk-types.js",
+    "ts:generate-types": "node ./scripts/type-generation/generate-TS-types.js",
```

Comment:
Should we use capitals in a case sensitive FS? I don't consider this a blocker, but it is an honest question.

Reply:
No, we shouldn't. I will update the filename.